manifest/com.Kannya/Console/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Console",
+	"id": "com.Kannya.Console",
+	"description": "A simple but very useful console mod for RML",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/KannyaResonite/Console",
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/KannyaResonite/Console/releases/tag/1.0.0",
+			"artifacts": [{
```

Comment:
Saw that you note prefixing the mod name with ! in your release, you can specify a `filename` for the artifact so that tools that download from the manifest will set that name for the file.